ls1intum/Artemis
# [Exam] Exam  & StudentExam View

### Checklist

### Motivation and Context
We had not yet implemented the Exam View page. 
The StudentExam View page had disabled buttons which were not needed. It also had todos for displaying the reviewer and results.

### Description
- [x] Implement the Exam View page
- [x] Remove unused buttons for the StudentExam View page 
- [x] Add results and reviewer to the StudentExam View page

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Administration
3. Create an exam
4. Click on the ID of the exam. All details should be shown correctly, all buttons should work as expected. Hint: the student exam button is only available to instructors.
5. Conduct the exam and review the submissions
6. Go to StudentExams and click on View for one of the student exams
7. The results should be shown correctly. For automatically assessed results, no reviewer is shown

### Screenshots
Student View
![image](https://user-images.githubusercontent.com/15110960/90037411-7867f100-dcc4-11ea-820e-ffe671b23f7d.png)
Exam View
![image](https://user-images.githubusercontent.com/15110960/90037676-c4b33100-dcc4-11ea-835e-7815cc6b0736.png)
